config/methods.config

```
@@ -18,14 +20,34 @@ methods {
                 if (params.samples_to_process.any { it.orig_id == sm_tags[0] }) {
                     throw new Exception("Sample ${sm_tags[0]} was found in multiple BAMs. Please provide only one BAM per sample")
                 }
-                new_sm_tag = methods.sanitize_uclahs_cds_id(sm_tags[0])
+                def new_sm_tag = methods.sanitize_uclahs_cds_id(sm_tags[0])
                 params.samples_to_process.add([
-                    'orig_id': sm_tags[0],
-                    'id': new_sm_tag,
                     'path': bam_path,
```

```suggestion
                    'bam': bam_path,
```
In a subsequent PR I change this 'path' to 'bam' throughout the repo.